packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1198,5 +1199,9 @@ export const METRICS_OPERATIONAL_MEASURES = {
   UNMUTE_VIDEO_FAILURE: 'js_sdk_unmute_video_failures',
   ROAP_ANSWER_FAILURE: 'js_sdk_roap_answer_failures',
   ROAP_GLARE_CONDITION: 'js_sdk_roap_glar_condition',
-  PEERCONNECTION_FAILURE: 'js_sdk_peerConnection_failures'
+  PEERCONNECTION_FAILURE: 'js_sdk_peerConnection_failures',
+  CONNECTION_FAILURE: 'js_sdk_connection_failure',
```

Comment:
For consistency, the measurement name should be plural. Same for the other new metrics
```suggestion
  CONNECTION_FAILURE: 'js_sdk_connection_failures',
```